cheroot/server.py

```diff
@@ -973,9 +973,6 @@ def respond(self):
 
         self.server.gateway(self).respond()
 
-        if (self.ready and not self.sent_headers):
-            self.sent_headers = True
-            self.send_headers()
```

Comment:
Why completely remove this? How will it affect non-WSGI plain HTTP server?

Reply:
I though that adding it to `self.server.gateway(self).respond()` would cover it

I suppose I can add it back to ensure we are always sending the headers in case the `gateway` is not sending the headers.